src/components/badge/index.js

```diff
@@ -35,7 +45,7 @@ export default class Badge extends BaseComponent {
     /**
      * the badge size (default, small)
      */
-    size: PropTypes.oneOf(['default', 'small', 'pimpleBig', 'pimpleSmall']),
+    size: PropTypes.oneOf(['default', 'small', 'pimpleSmall', 'pimpleBig', 'pimpleHuge']),
```

Comment:
consider using  `PropTypes.oneOf(Object.keys(BADGE_SIZE))` here, do we dont have to update this propType all the time. 
(btw, here it's right to use `Object.keys`)